doc/user_guide/install.rst

```diff
@@ -31,188 +31,144 @@ scientific Python packages.
+  on `traitsui <http://docs.enthought.com/traitsui/>`_,
+* ``mrcz`` to install the mrcz plugin,
+* ``speed`` install optional libraries that speed up some functionalities,
+* ``tests`` to install required libraries to run HyperSpy's unit tests,
+* ``build-doc`` to install required libraries to build HyperSpy's documentation,
+* ``dev`` to install all the above,
+* ``all`` to install all the above expect the development requirements
+  (``tests`` and ``build-doc``).
 
 For example:
 
 .. code-block:: bash
 
     $ pip install hyperspy[learning, gui-jupyter]
 
-See also :ref:`install-dependencies`.
-
-Finally, be aware that HyperSpy depends on a
-number of libraries that usually need to be compiled and therefore installing
-HyperSpy may require development tools. If the above does not work for you
-remember that the easiest way to install HyperSpy is
-:ref:`using Anaconda <quick-anaconda-install>`.
+Finally, be aware that HyperSpy depends on a number of libraries that usually 
+need to be compiled and therefore installing HyperSpy may require development
+tools installed in the system. If the above does not work for you remember that
+the easiest way to install HyperSpy is
+:ref:`using Anaconda or Miniconda <anaconda-install>`.
 
+.. _install-dev:
 
-.. _install-binary:
+Install development version
+---------------------------
 
-Install from a binary
----------------------
+Clone the hyperspy repository
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We provide  binary distributions for Windows (`see the
-Downloads section of the website <http://hyperspy.org/download.html>`_). To
-install easily in other platforms see :ref:`install-with-python-installers`
+To get the development version from our git repository you need to install `git
+<http://git-scm.com//>`_. Then just do:
 
+.. code-block:: bash
 
-.. _install-source:
+    $ git clone https://github.com/hyperspy/hyperspy.git
 
-Install from source
--------------------
+.. Warning::
 
-.. _install-released-source:
+    When running hyperspy from a development version, it can happen that the
+    dependency requirement changes in which you will need to keep this
+    this requirement up to date (check dependency requirement in ``setup.py``)
+    or run again the installation in development mode using ``pip`` as explained
+    below.
 
-Released version
-^^^^^^^^^^^^^^^^
+Installation in a Anaconda/Minconda distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To install from source grab a tar.gz release and in Linux/Mac (requires to
-:ref:`install-dependencies` manually):
+Install the runtime and development dependencies requirements using conda:
 
 .. code-block:: bash
 
-    $ tar -xzf hyperspy.tar.gz
-    $ cd hyperspy
-    $ python setup.py install
+    $ conda install hyperspy-base -c conda-forge --only-deps
+    $ conda install hyperspy-dev -c conda-forge
 
-You can also use a Python installer, e.g.
+The package ``hyperspy-dev`` will install the development dependencies required
```

Comment:
Can we add a similar sentence describing hyperspy-base, here? It's unclear whether hyperspy-base refers to a hyperspy install with only the strictly required dependencies (like `pip install hyperspy`) or whether it installs everything except the developer-dependencies (like I think it does?).